nixos/modules/services/desktops/pipewire.nix

```diff
@@ -93,11 +93,11 @@ in {
     assertions = [
       {
         assertion = cfg.pulse.enable -> !config.hardware.pulseaudio.enable;
-        message = "PipeWire based PulseAudio server emulation replaces PulseAudio";
+        message = "PipeWire based PulseAudio server emulation replaces PulseAudio. Disable `hardware.pulseaudio.enable`";
```

Comment:
Do I need to disable something or is something for me disabled?

Reply:
Depending on the desktop environment you're using, you might just need to remove `hardware.pulseaudio.enable = true` since it's disabled by default, but if you're using something like gnome, you will have to explicitly disable it, because it automatically enables it: https://github.com/NixOS/nixpkgs/blob/4746e1717f082663813861046c2774f395402bda/nixos/modules/services/x11/desktop-managers/gnome3.nix#L270

Do you think there's a clearer way to state this?

Reply:
```
PipeWire server emulates and replaces PulseAudio. This option forces `hardware.pulseaudio.enable` to false.
```

Reply:
This assertion require user action, the pipewire module doesn't automatically set the options to false (but maybe it should @jtojnar @worldofpeace?)

Reply:
You see the problem? The message is unclear.

```
PipeWire server emulates and replaces PulseAudio. This option requires `hardware.pulseaudio.enable` to be set to false.
``

Reply:
Yep, thank you. I was also thinking of changing it to something like that to make it clearer. The only possible confusion is that in some cases you don't have to explicitly set `hardware.pulseaudio.enable` to false because in some environments it defauls to false, but this is clearer than the original message.